### PR TITLE
Introduce distrib_cc_library

### PR DESCRIPTION
**Pull request by meteorcloudy**
Similar to distrib_java_import, this macro rule is for redirecting
Bazel's cc dependencies to Debian's system libraries.

Related https://github.com/bazelbuild/bazel/issues/9408

### CODE REVIEW

File: tools/distributions/distribution_rules.bzl [hunk 1]
```diff
@@ -14,11 +14,11 @@
 
 load("@rules_java//java:defs.bzl", _java_import = "java_import")
 
-def distrib_java_import(name, enable_distributions = [], **kwargs):
+def distrib_java_import(name, visibility = None, enable_distributions = [], **kwargs):
```
Comment (meisterT) [inline tools/distributions/distribution_rules.bzl:17]: please split this off into a separate commit

Comment (meteorcloudy) [reply tools/distributions/distribution_rules.bzl]: Done, I sent another CL to fix the visibility problem.

**Comment by meteorcloudy** [comment]
FYI @olekw
